paparazzi-agent/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'java-library'
+apply plugin: 'org.jetbrains.kotlin.jvm'
+
+dependencies {
+  implementation deps.kotlin.stdlib.jdk8
+
+  implementation 'net.bytebuddy:byte-buddy:1.10.10'
+  implementation 'net.bytebuddy:byte-buddy-agent:1.9.0'
```

Comment:
Move to dep map?

Also why different versions?

Reply:
Ah, forgot about this.  Good catch.

No good reason.  I think I copy-pasted from different sites and forgot to align 'em.

Both now fixed.